static/app/components/charts/baseChart.tsx

```diff
@@ -704,6 +704,11 @@ const getTooltipStyles = (p: {theme: Theme}) => css`
     justify-content: space-between;
     align-items: baseline;
   }
+  .code-no-margin {
```

Comment:
can we add tooltip here just to keep things labled
```suggestion
  .tooltip-code-no-margin {
```